src/test/test_type_to_num.cpp

```
@@ -0,0 +1,70 @@
+
+#include <catch2/catch_all.hpp>
+
+#include "arith/dsl_syntax.hpp"
+#include "crab/type_to_num.hpp"
+
+using namespace prevail;
+
+static constexpr Reg r0{0};
+static constexpr Reg r1{1};
+
+TEST_CASE("forget_type_dependent_values preserves type and universal value", "[type_to_num]") {
```

Clang (14.0.6)

[warning] variable 'TEST_CASE' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)